examples/gizmos/2d_gizmos.rs

```diff
@@ -87,6 +87,8 @@ fn draw_example_collection(
         Vec2::from_angle(sin / -10. + PI / 2.) * 50.,
         YELLOW,
     );
+
+    gizmos.axes_2d(Transform::from_translation(Vec3::new(sin, 0.0, 0.0)), 25.0)
```

Comment:
this example is already quite crowded. this will also put it on the grid gizmo already present, making it hard to read